_posts/2022-11-07-changelog-182.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #182"
+date: 2022-11-07 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#9650] Increase the minimal supported version of Rust to 1.56
+
+* Procedural macro support improvements:
+  * [#9628] Enable the expansion of [function-like procedural macros] in the [nightly builds].
+    This feature can be controlled via `org.rust.macros.proc.function-like` option in `Experimental Features` dialog.
+    For more information, check the [implementation PR](https://github.com/intellij-rust/intellij-rust/pull/6564) and the corresponding [tracking issue](https://github.com/intellij-rust/intellij-rust/issues/6908).
+
+  * [#9526] Insert brackets of the proper form (`()`, `[]` or `{}`) when completing procedural macro calls
+
+  * [#9614] Fix the expansion of [attribute] and [derive] procedural macros inside functions
+
+  * [#9559] Fix duplicate completion for already imported [attribute] procedural macros
+
+* [#9105] Allow to run a specific [documentation test](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html) [all the ways](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-testing.html) you can run a usual test (by [@Kobzol])
+TODO: gif
+
+* [#9349] Decorate MSVC type names when rendering values in the debugger on Windows. Now the Rust type names containing MSVC-specific wrappers are rendered properly, e.g. `tuple$<Foo, ref$<Bar>>` is replaced by `(Foo, &Bar)`.
+  This feature can be disabled via the `Decorate MSVC type names` checkbox in the `File | Settings | Build, Execution, Deployment | Debugger | Data Views | Rust` settings
+TODO: image
+
+* [#7134] Add a quick-fix to generate missing trait member definitions in trait implementations (by [@Sejsel])
+TODO: gif
+
+* [#8755] Add basic parsing support for `let_chains` (by [@Kobzol])
```

Comment:
It seems the support is not so basic

Also, I'd move it at the top of the section